### PR TITLE
[AAE-11496] Move 'content-plugin' to projects folder as 'aca-content'

### PR DESCRIPTION
**Pull request by BSekula**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/AAE-11496

**What is the new behaviour?**
Moved `content-plugin` to `projects/aca-content`

Before merging this one, we need to merge `alfresco-apps` PR - https://github.com/Alfresco/alfresco-apps/pull/4675

**Does this PR introduce a breaking change?** (check one with "x")

> - [x] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/aca-content/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "aca-content",
```
Comment (DenysVuika) [inline projects/aca-content/package.json:2]: I think this file is not needed, as the library is being mapped in tsconfig

File: projects/aca-content/src/lib/aca-content.component.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (DenysVuika) [inline projects/aca-content/src/lib/aca-content.component.ts:1]: not needed

**Comment by DenysVuika** [comment]
Please don't forget to include the testing into the Travis config

File: projects/aca-content/src/lib/aca-content.component.ts (hunk 2 above)
Comment (BSekula) [reply projects/aca-content/src/lib/aca-content.component.ts:1]: Hi Denys, it is a draft right now :) have you been still notify? My plan is to divide this PR to at least 2 commits.

One is to create `libraby` and everythig what is needed to make it work, second part is going to be moving the `content` to the library, so the number of moved files won't hide the changes

File: projects/aca-content/package.json (hunk 1 above)
Comment (BSekula) [reply projects/aca-content/package.json:2]: We are going to make a npm package out of it, so I believe that we are going to need it

File: projects/aca-content/src/lib/content-services-extension.module.ts [hunk 3]
```diff
@@ -140,7 +138,7 @@ registerLocaleData(localeSv);
 @NgModule({
   imports: [
     BrowserModule,
-    environment.e2e ? NoopAnimationsModule : BrowserAnimationsModule,
+    // environment.e2e ? NoopAnimationsModule : BrowserAnimationsModule,
```
Comment (DenysVuika) [inline projects/aca-content/src/lib/content-services-extension.module.ts:141]: commented out code

File: projects/aca-content/src/lib/store/app-store.module.ts [hunk 4]
```diff
@@ -28,8 +28,8 @@ import { StoreModule } from '@ngrx/store';
 import { appReducer } from './reducers/app.reducer';
 import { StoreRouterConnectingModule, FullRouterStateSerializer } from '@ngrx/router-store';
 import { EffectsModule } from '@ngrx/effects';
-import { environment } from '../../../environments/environment';
-import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+// import { environment } from '../../../environments/environment';
```
Comment (DenysVuika) [inline projects/aca-content/src/lib/store/app-store.module.ts:31]: commented out code

File: app/src/styles.scss [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
-@import 'app/content-plugin/ui/application';
-@import 'app/content-plugin/ui/variables/font-family';
+@import 'projects/aca-content/src/lib/ui/application';
```
Comment (DenysVuika) [inline app/src/styles.scss:2]: this should be a reference to the NPM project? like we import ADF?

**Review by DenysVuika — COMMENTED** [review]
I know the work is in progress, left a few comments about code to pay attention to

File: projects/aca-content/src/lib/content-services-extension.module.ts [hunk 6]
```diff
@@ -25,7 +25,7 @@
 
 import { BrowserModule, HammerModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { BrowserAnimationsModule, NoopAnimationsModule } from '@angular/platform-browser/animations';
+// import { BrowserAnimationsModule, NoopAnimationsModule } from '@angular/platform-browser/animations';
```
[the remark this answers is not in the document]
Comment (BSekula) [reply projects/aca-content/src/lib/content-services-extension.module.ts]: removed

File: projects/aca-content/src/lib/content-services-extension.module.ts [hunk 7]
```diff
@@ -75,7 +73,7 @@ import { FavoritesComponent } from './components/favorites/favorites.component';
 import { RecentFilesComponent } from './components/recent-files/recent-files.component';
 import { SharedFilesComponent } from './components/shared-files/shared-files.component';
 import { CreateFromTemplateDialogComponent } from './dialogs/node-template/create-from-template.dialog';
-import { environment } from '../../environments/environment';
+// import { environment } from '../../environments/environment';
```
[the remark this answers is not in the document]
Comment (BSekula) [reply projects/aca-content/src/lib/content-services-extension.module.ts]: removed

File: projects/aca-content/src/lib/content-services-extension.module.ts (hunk 3 above)
Comment (BSekula) [reply projects/aca-content/src/lib/content-services-extension.module.ts]: removed

File: projects/aca-content/src/lib/store/app-store.module.ts (hunk 4 above)
Comment (BSekula) [reply projects/aca-content/src/lib/store/app-store.module.ts]: removed

File: app/src/styles.scss (hunk 5 above)
Comment (BSekula) [reply app/src/styles.scss:2]: moved to angular.json.

```
"styles": [
              ...
              "projects/aca-content/src/lib/ui/application.scss"
            ],
```

I think we are going to import it like from adf when we are going to install it in different repository, currently we are in the same repo (I am going to create a different PR in order to build it).

File: projects/aca-content/src/lib/components/search/search-libraries-results/search-libraries-results.component.scss [hunk 8]
```diff
@@ -1,4 +1,4 @@
-@import 'mixins';
+@import 'projects/aca-content/src/lib/ui/mixins';
```
Comment (BSekula) [inline projects/aca-content/src/lib/components/search/search-libraries-results/search-libraries-results.component.scss:1]: I cannot find any working solution to import it as just `mixin` 

@DenysVuika maybe you would know how to handle this issue?

File: projects/aca-content/src/lib/components/search/search-results/search-results.component.scss [hunk 9]
```diff
@@ -1,4 +1,4 @@
-@import 'mixins';
+@import 'projects/aca-content/src/lib/ui/mixins';
```
Comment (BSekula) [inline projects/aca-content/src/lib/components/search/search-results/search-results.component.scss:1]: The same as [above](https://github.com/Alfresco/alfresco-content-app/pull/2817/files#r1052649093)

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**